src/module-elasticsuite-swatches/Block/Navigation/Renderer/Swatches/RenderLayered.php

```diff
@@ -37,15 +40,20 @@ public function getSwatchData(): array
         }
 
         $attributeOptions = [];
+        // Collect parameter labels in the expected order
         $attributeOptionsSort = [];
-
+        // Build an array whose keys are the attribute option label and not option id as in the native method
```

Comment:
A full stop (".") is required at the end of the sentence to avoid a phpcs error.